test/rootfs/setup.sh

```diff
@@ -227,6 +227,13 @@ git clone https://github.com/sstephenson/bats.git "${tmpdir}/bats"
 curl -fsLo "/usr/local/bin/jq" "http://stedolan.github.io/jq/download/linux64/jq"
 chmod +x "/usr/local/bin/jq"
 
+# install protobuf compiler
+curl -sL https://github.com/google/protobuf/releases/download/v3.0.0/protoc-3.0.0-linux-x86_64.zip > ~/protoc.zip
+trap "rm ~/protoc.zip" EXIT
```

Comment:
This trap will overwrite the earlier traps (so we are already leaking some temp files as we have multiple traps). We should just create a tmpdir at the top of the script with a single trap to remove it then use it throughout the script.